imports/schemas/userEvent.coffee

```diff
@@ -4,22 +4,32 @@ userEventSchema = new SimpleSchema(
     optional: true
   articleCount:
     type: Number
+    optional: true
```

Comment:
The default is for them to be required. The keys are not required by the meteor method using this schema for validation.